releases/handbook.md

```diff
@@ -210,149 +88,44 @@ Criteria for timeline that the team needs to consider
-
-After the 3 weeks pass, for this phase, the release process will not further wait for the distributions to be ready.
-
-**Actions for other WGs:**
-
-- Evaluate which of the reported issues should be release blocking
-- Work on providing bug fixes for release blocking issues
-- Create a final git tag. It should be stable (not RC) and include fixes for release blocking issues found during this time
-
-**Actions for Manifests WG:**
-
-- (Optional) Create a **vX.Y-rc.3** tag, depending on the number of created issues and bug fixes
-
-**Actions for Release Team:**
-
-- Work with the WG to build the release slides
+- Work with the WG to build the release notes and slides
 - Start creating the draft for the official blog post and collating information from the Working Groups
     - (Optional but encouraged) Working Groups start drafting WG-specific blog
         posts, deep diving into their respective areas
 - Preparation for social media posts can start at the beginning of this phase
-- Release Manager: List the features, and ideally with docs, that made it into the release
-
-**Success Criteria:** Thumbs up from distribution representatives when their testing is complete.
-
-### Release
-
-We made it!
-
-**Actions for Manifests WG:**
-
-- Create the final **vX.Y** tag, pointing to the latest commit of the release branch
-- Cut a new GitHub release in <https://github.com/kubeflow/kubeflow>
-- Cut a new GitHub release in <https://github.com/kubeflow/manifests>
-
-**Actions for Release Team:**
-
+- Release Manager: List the features, and ideally with documentation, that made it into the release
 - Publish release blog post
 - (Optional but encouraged) Working Groups publish individual deep dive blog posts on features or other work they’d like to see highlighted.
 - Publish social media posts
 - Send [release announcement](https://groups.google.com/g/kubeflow-discuss/c/qDRvrLPHU70/m/ORKN14DzCQAJ) to kubeflow-discuss
-- Coordinate with the liaison for each WG to record the highlight video as part of the release meeting, or ask the liaison to share a prerecorded .mp4
+
+**Success Criteria:**
+- Documentation for this release completed with minimum following pages updated and a [new version
+in the website is cut](https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#version-the-website).
+- [Installing Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/)
+- [Release Page](https://www.kubeflow.org/docs/releases/)
+- [Distributions](https://www.kubeflow.org/docs/distributions/) and related pages underneath
 
 ## Post Release
 
 ### Patch Release
 Planning for first patch release begins. The importance of bugs is left to the
-judgement of the Working Group's tech leads and the Release Manager to decide.
```

Comment:
who's making the decision on patches if it's not the release manager with WG TL or why this section was removed.